examples/grid_example/public/app.tsx

```diff
@@ -311,7 +313,7 @@ const layoutStyles = ({ euiTheme }: UseEuiTheme) => {
       transition: `opacity 100ms linear`,
     },
     // styling for panel resize handle
-    '.kbnGridPanel--resizeHandle': {
+    '.kbnGridPanel__resizeHandle': {
```

Comment:
I changed the name to better match the BEM naming, but if there's a reason to leave it as it is, I can revert it!

Reply:
If we want to standardize on BEM, then I think we should do that as a separate PR - this other naming convention  is used a lot in this package, and I'd rather stay consistent :)

Reply:
Ok, reverted! https://github.com/elastic/kibana/pull/208286/commits/3caed27963bdcf1b4a1db190048ef8b133a7a6a7

Reply:
ooooops sorry! 🤦🏼‍♀️